### PR TITLE
Registration table "see more" link (aka 'patient j must die')

### PR DESCRIPTION
**Pull request by erictzeng**
The registration table "see more" link works now.

By default, views are limited to the last 10 entries. Clicking on "see more" reveals all of them by appending ".full" to the URL:

`/registration/registrars/1` becomes `/registration/registrars/1.full`

IT'S A CUCUMBER

FRIGGIN' PATIENT J

### CODE REVIEW

File: app/controllers/registrars_controller.rb [hunk 1]
```diff
@@ -1,23 +1,26 @@
...
-
+    @name = registrar.name
+    index_or_show registrar.registrations
   end
 
   protected
+  def index_or_show(regs)
+    @num_today = regs.for_day(Time.now).count
+    @average_time = average_time(regs)
+    @full = (params[:format] == 'full')
```
Comment (seshness) [inline app/controllers/registrars_controller.rb]: Is it really a different format? imho the URL ought to look like `/registration/registrars/1/full`

Comment (seshness) [inline app/controllers/registrars_controller.rb]: I would think something an alternative format if it returned the same information but with different "packaging".
Accessing the subresource `full` gets you more information, as opposed to returning the same information in a different form.

Comment (erictzeng) [inline app/controllers/registrars_controller.rb]: You're probably right, I just didn't have enough time to wade through routes hell to fix it. I'll fix if I have time before the deadline...

Comment (seshness) [inline app/controllers/registrars_controller.rb]: okay, merge for now. fix after deadline

Comment (erictzeng) [inline app/controllers/registrars_controller.rb]: 180339f

**Pull request merged**